packages/combo-box/src/vaadin-combo-box-overlay-mixin.js

```diff
@@ -22,21 +22,6 @@ export const ComboBoxOverlayMixin = (superClass) =>
       this.requiredVerticalSpace = 200;
     }
 
-    /** @protected */
-    connectedCallback() {
-      super.connectedCallback();
-
-      const hostDir = this._getHostDir();
```

Comment:
This stopped working because with native `popover`, overlay no longer gets re-connected while opening.
So I replaced it with `.dir="${this.dir}"` for combo-box & MSCB, and `dir="ltr"` for the time-picker.